src/stores/widgets/WidgetLayoutStore.ts

```diff
@@ -201,12 +202,21 @@ export class WidgetLayoutStore extends ReadyWatchingStore {
         // function will go into the right container.
```

Comment:
The above comments about only two containers (here and other places in the file) need some updates.